pyfar/plot/line.py

```diff
@@ -193,6 +193,8 @@ def phase(signal, deg=False, unwrap=False, freq_scale='log', ax=None,
         ``style = {'axes.facecolor':'black'}``. Pass an empty dictionary
         ``style = {}`` to use the currently active plotstyle. The default is
         ``light``.
+    side : str
+        ``left`` or ``right`` to specify which side of a spectrum is plotted.
```

Comment:
I would suggest to give a little more information and also mention the default here and everywhere else:

```suggestion
    side : str, optional
        ``'right'`` to plot the right-sided spectrum containing the positive
        frequencies, or ``'left'``to plot the left-sided spectrum containing
        the negative frequencies (only possible for complex Signals). The
        default is ``'right'``.
```